File: api/SprintoApi/SprintoApi/Models/Projects.cs
using System.ComponentModel.DataAnnotations;

namespace SprintoApi.Models
{
    public class Projects
    {
        [Key]
        public int id { get; set; }
        public string title { get; set; }
        public string description { get; set; }
        public DateTime created_date { get; set; }
        public int created_by { get; set; }

    }
    public class ProjectsDto
    {
        public int created_by { get; set; }

    }

}

File: app/lib/screens/home/models/projects.dart
class Projects {
  int id;
  String title;
  String description;
  DateTime createdDate;
  int createdBy;

  Projects({
    required this.id,
    required this.title,
    required this.description,
    required this.createdDate,
    required this.createdBy,
  });

  factory Projects.fromJson(Map<String, dynamic> json) => Projects(
    id: json["id"],
    title: json["title"],
    description: json["description"],
    createdDate: DateTime.parse(json["created_date"]),
    createdBy: json["created_by"],
  );

  Map<String, dynamic> toJson() => {
    "id": id,
    "title": title,
    "description": description,
    "created_date": createdDate.toIso8601String(),
    "created_by": createdBy,
  };
}
